frontend/src/components/marketDetails/MarketDetailsLayout.jsx

```diff
@@ -1,75 +1,105 @@
+          {market.questionTitle}
+        </h1>
+        <div className='flex items-center space-x-2 text-sm text-gray-400'>
+          <Link
+            to={`/user/${market.creatorUsername}`}
+            className='hover:text-blue-400 transition-colors duration-200'
+          >
+            <span role='img' aria-label='Creator'>
+              {creator.personalEmoji}
+            </span>
+            @{market.creatorUsername}
+          </Link>
+          <span>•</span>
+          <span>🪙 {currentProbability.toFixed(2)}</span>
+        </div>
+      </div>
+
+      <div className='mb-4'>
+        <MarketChart
+          data={probabilityChanges}
+          currentProbability={currentProbability}
+          title='Probability Changes'
+          className='w-full'
+        />
+      </div>
+
+      <div className='mb-4'>
+        <button
+          onClick={() => setShowFullDescription(!showFullDescription)}
+          className='w-full py-2 bg-gray-700 hover:bg-gray-600 transition-colors duration-200 rounded-lg text-center text-sm'
+        >
+          {showFullDescription ? 'Hide Description' : 'Show Full Description'}
+        </button>
+      </div>
 
-function MarketDetailsTable({ market, creator, numUsers, totalVolume, currentProbability, probabilityChanges }) {
-    const [showFullDescription, setShowFullDescription] = useState(false);
+      <div className='mb-4 bg-gray-800 p-4 rounded-lg'>
+        <p
+          className={`text-sm break-words whitespace-pre-wrap ${
+            showFullDescription
+              ? ''
+              : 'sm:max-h-24 h-16 overflow-y-auto sm:overflow-hidden'
+          }`}
+          style={{
+            wordBreak: 'break-word',
+            overflowWrap: 'break-word',
+            hyphens: 'auto',
+          }}
+        >
+          {market.description}
+        </p>
+      </div>
 
-    // Function to truncate the description to 50 characters
-    const truncateDescription = (description) => {
-        if (description.length > 50) {
-            return `${description.substring(0, 100)}...`;
-        }
-        return description;
-    };
+      <div className='grid grid-cols-2 sm:grid-cols-4 gap-2 text-center mb-4'>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Users</div>
+          <div className='text-sm font-semibold'>👤 {numUsers}</div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Volume</div>
+          <div className='text-sm font-semibold'>
+            📊 {totalVolume.toFixed(2)}
+          </div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Comments</div>
+          <div className='text-sm font-semibold'>💬 0</div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Closes</div>
```

Comment:
Though the MarketTables seem to show, "Closed" when the close date > present date, it still shows as just a date on the MarketDetails page here.

![image](https://github.com/user-attachments/assets/d115072f-2a36-4c42-9fcb-74415585c660)

Reply:
On it

Reply:
Fixed here https://github.com/openpredictionmarkets/socialpredict/pull/242/commits/0e338653ee4687f318c6660a607dfc56ddecfe9d